docs/how-to/uc-image-creation.rst

```diff
@@ -139,6 +139,12 @@ This downloads all the snaps specified in the model assertion and builds an imag
 
 ✅ The image file is now ready to be flashed on a medium to create a bootable drive with the Ubuntu Core installer!
 
+Once booted, the kernel parameters can be verified by looking into ``/proc/cmdline``:
+
+.. code-block:: console
+
+    $ cat /proc/cmdline
+    snapd_recovery_mode=run console=ttyS0,115200n8 console=tty1 panic=-1 nohz=on nohz_full=2-N irqaffinity=0-1
```

Comment:
Looks good! In addition, you can add `isolcpus` to the list of command-line parameters and select the CPUs you want to isolate from the kernel process scheduler.

Reply:
Yes, but why?

I have used the same examples as in the other how-to:
https://github.com/canonical/real-time-ubuntu-docs/blob/a173b1c79cbc0d568ac100eb787e2317d126e622/docs/how-to/uc-boot-parameters.rst?plain=1#L38-L49

Reply:
Because `isolcpus` is a perfect use case for real-time kernel tuning, it provides exactly what a CPU destined to handle real-time workloads needs: a processing unit isolated from [system noise](https://docs.kernel.org/trace/osnoise-tracer.html).

> I have used the same examples as in the other how-to:

Yeah, it would be good to have `isolcpus` on those examples as well.

Reply:
I don't disagree that `isolcpus` is an important parameter. But the how-tos aren't about the parameters themselves, instead they serve to explain the method for setting any parameter. We have a reference listing all the possibilities, I will link to that.